tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -99,4 +103,115 @@ def test_max_login_failure(monkeypatch):
                     "Please contact DATABroker@fiscal.treasury.gov to obtain access."
 
     # Not in cgac group
-    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
\ No newline at end of file
+    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
+
+
+def test_list_submissions_success(database, job_constants, monkeypatch):
```

Comment:
I can break it down into multiple tests and mimic `error_rows` and have something similar that adds and cleans up in one go. But the clean up would still need to happen explicitly.